cluster/addons/ip-masq-agent/ip-masq-agent.yaml

```diff
@@ -0,0 +1,39 @@
+# https://github.com/kubernetes-incubator/ip-masq-agent/blob/v2.0.0/README.md
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: ip-masq-agent
+  namespace: kube-system
+  labels:
+    kubernetes.io/cluster-service: "true"
```

Comment:
I'm not sure you need this label, https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/addon-manager/README.md seems to say that it's deprecated.